terraform/app_service.tf

```diff
@@ -53,10 +53,87 @@ resource "azurerm_linux_web_app" "main" {
     mount_path   = "/home/calitp/app/config"
   }
 
+  app_settings = {
+    # app setting used solely for refreshing secrets - see https://github.com/MicrosoftDocs/azure-docs/issues/79855#issuecomment-1265664801
+    "change_me_to_refresh_secrets" = "change me in the portal to refresh all secrets",
+
+    "DOCKER_ENABLE_CI"                    = "true",
+    "DOCKER_REGISTRY_SERVER_URL"          = "https://ghcr.io/",
+    "WEBSITE_HTTPLOGGING_RETENTION_DAYS"  = "99999",
+    "WEBSITE_TIME_ZONE"                   = "America/Los_Angeles",
+    "WEBSITES_ENABLE_APP_SERVICE_STORAGE" = "false",
+    "WEBSITES_PORT"                       = "8000",
+
+    "ANALYTICS_KEY" = local.is_dev ? null : "@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=analytics-key)",
```

Comment:
Is there a way to create a local variable for the repeated part of this string? E.g. 

`@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=`

And then fill in the secret name in each case?

Reply:
Good idea. Done in [00287f5](https://github.com/cal-itp/benefits/pull/1294/commits/00287f5e49d534a082ce321c83b3c539d348ea0d)

Reply:
Done in [5cdc6a9](https://github.com/cal-itp/benefits/pull/1294/commits/5cdc6a9b589b13e85153be27f73335eff522848a)

Reply:
Ah, so much cleaner! Diffs will be easier now too I think